jax/experimental/jax2tf/jax_export.py

```diff
@@ -955,12 +990,14 @@ def _call_exported_impl(*args, exported: Exported):
 def _call_exported_lowering(ctx: mlir.LoweringRuleContext, *args,
                             platform: str,
                             exported: Exported):
-  if (platform != exported.lowering_platform and
+  # TODO: implement true multi-platform lowering for call_exported
+  if (platform not in exported.lowering_platforms and
       DisabledSafetyCheck.platform() not in exported.disabled_checks):
     raise ValueError(
         f"The exported function '{exported.fun_name}' was lowered for "
         f"platform '{exported.lowering_platform}' but it is used "
```

Comment:
nit: we also need update the error msg here "pltforms '{exported.lowering_platforms}'" to keep consistency.